doc/application/application.rst

```diff
@@ -223,105 +621,142 @@ Below is an example CMakeList.txt:
 Application Configuration
 *************************
 
-The application's kernel is configured using a set of configuration options
+The application's kernel is configured using a set of options
 that can be customized for application-specific purposes.
 The Zephyr build system takes a configuration option's value from the first
 source in which it is specified.
 
-The available sources are (in order):
+An option's value is taken from the following available sources, in order:
```

Comment:
merge this with the previous sentence:

      ... it is specifiied, taken from the following available sources, in order:

Reply:
That's better. Will do, thanks.